src/archiefvernietigingscomponent/destruction/tasks.py

```diff
@@ -49,6 +55,35 @@ def process_destruction_list(list_id):
     chain(chunk_tasks.group(), notify_task)()
 
 
+@app.task
+def check_if_reviewers_need_reminder():
+    destruction_lists = DestructionList.objects.all()
```

Comment:
The query you provided above returned an empty queryset - surely it's easier to simply do a check to see if the assignees list query set is empty before proceeding?

Reply:
The query above would return an empty query set if you haven't modified the tests yet (since as I mention below, the tests don't set the `destruction_list.assignee`). 

The advantage of the query above is that it should make one query to the database without needing to do any additional checks, you can just send the emails to the users related to the assignees retrieved.